src/NuGet/Microsoft.NETCore.Compilers.nuspec

```diff
@@ -49,6 +52,7 @@
     <file src="Exes/VBCSCompiler/netcoreapp2.0/publish/VBCSCompiler.runtimeconfig.json" target="tools/bincore" />
 
     <!-- References that are either not in the target framework or are a higher version -->
+    <!-- N.B.: The backslashes below cannot be replaced with forward slashes. -->
```

Comment:
I don't want to ask but I have to ... why?

Reply:
It looks like NuGet has different behavior -- the backslashes are interpreted as "everything in this directory" while forward slashes are "the entire directory tree, including parent directories"